shared/networking/1.X.md

```diff
@@ -0,0 +1,159 @@
+
+### Manually Editing Config.json
+
+Currently this can be done by editing the `config.json`. This file can be found in a partition called `resin-boot` on the SD card for most devices (on Resin OS versions before 1.2 it was `resin-conf` or `flash-conf`), except the Intel Edison. For the Intel Edison it can be found in in `resin-conf` once you have mounted the `config.img`.
+
+__Note:__ For the Beaglebone, VIA VAB-820, Intel NUC and the Intel Edison, you can only change the wifi configuration **before you provision** the device. Since these devices rely on burning the OS to internal media such as eMMC, trying to change these settings after provisioning will have no effect, you will need to reprovision the device.
+
+In the `config.json` file you will need to edit the section called `files` with whatever `SSID` and `passphrase` you require.
+
+```JSON
+"files": {
+    "network/settings": "[global]\nOfflineMode=false\n\n[WiFi]\nEnable=true\nTethering=false\n\n[Wired]\nEnable=true\nTethering=false\n\n[Bluetooth]\nEnable=true\nTethering=false",
+    "network/network.config": "[service_home_ethernet]\nType = ethernet\nNameservers = 8.8.8.8,8.8.4.4\n\n[service_home_wifi]\nType = wifi\nName = My_Wifi_Ssid\nPassphrase = my super secret wifi passphrase\nNameservers = 8.8.8.8,8.8.4.4"
+  }
+```
+__Note:__ Unfortunately this file is not nicely formatted and it requires that you use the `\n` to signify a newline.
+
+## 3G or Cellular Connections
+
+Currently 3G or Cellular modems are not supported out of the box, but they are easily setup on a resin.io device. During the setup of your cellular connection, you will need an ethernet or wifi connection to the device so that it can correctly provision and download a container with the necessary configurations.
+
+To get started with a cellular connection have a look at our blog post ["A guide to cellular connectivity on resin.io devices"](https://resin.io/blog/cellular-connectivity/) and the corresponding [github repository](https://github.com/resin-io-projects/cellular-modem.git).
+
+## Captive Portal Network Setup
+
+In some cases devices will need to be provisioned and sent out to sites where the exact network or wifi configuration is unknown. In these circumstances its often nice for a device to serve up an [captive portal][captive-portal-link] which allows an onsite user to configure the wifi for a device.
+
+If you require this functionality in your devices, you should have a look at our captive portal project called [Resin Wifi Connect][wifi-connect-link] and its associated [blog post][wifi-connect-blog]
+
+### Troubleshooting
+
+If you have issues connecting with the WiFi device, first check to ensure the
+SSID and passphrase are correct. If they are, try rebooting with an ethernet
+cable plugged in, then booting again with just WiFi.
+
+If neither of these approaches work, drop us a line at support@resin.io!
```

Comment:
Link to the forums instead of the support email? Or what's our current policy regarding support emails?